cli/cmd/plugin/unmanaged-cluster/test/e2e/utils/cli.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/onsi/ginkgo"
+)
+
+func InstallTCE() error {
+
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Println("error while getting current working directory", err)
+	}
+
+	indTCE := strings.LastIndex(wd, "community-edition")
+	topDirPath := wd[0 : indTCE+17]
+	err = os.Chdir(topDirPath)
+	if err != nil {
+		log.Println("error while changing directory to the top:", err)
+		return err
+	}
+
+	// installing dependencies i.e docker and kubectl
+	err = runDeployScript("hack/ensure-deps/check-docker.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	err = runDeployScript("hack/ensure-deps/ensure-kubectl.sh")
+	if err != nil {
+		log.Fatal(err)
+		return err
+	}
+
+	// build.sh is creating a release and installing TCE for latest code
+	return runDeployScript("test/build-tce.sh")
```

Comment:
Is this supposed to be [download-or-build-tce.sh](https://github.com/vmware-tanzu/community-edition/blob/main/test/download-or-build-tce.sh)?

Reply:
I thought to include all the commits not just nightly build if i have understood the above script correct which is first looking for nightly build.

Reply:
Sure changing it to [download-or-build-tce.sh](https://github.com/vmware-tanzu/community-edition/blob/main/test/download-or-build-tce.sh)